config/config.js

```diff
@@ -2,14 +2,26 @@ const os = require('os')
 const fs = require('fs')
 const path = require('path')
 const yaml = require('js-yaml')
+const chalk = require('chalk')
 
 const defaultFullPath = path.join(os.homedir(), '.eureka', 'eureka.config.yaml')
 const filename = process.env.EUREKA_CONFIG || defaultFullPath
 
-if (!fs.existsSync(filename)) {
-  throw new Error(`Missing Eureka configuration file in ${filename}. Please contact Eureka team for on-boarding instructions`)
-}
-
-const data = yaml.safeLoad(fs.readFileSync(filename, 'utf8'))
+try {
+  if (!fs.existsSync(filename)) {
+    throw new Error(`Missing Eureka configuration file in ${filename}. Please contact Eureka team for on-boarding instructions`)
+  }
 
-module.exports = data
+  const data = yaml.safeLoad(fs.readFileSync(filename, 'utf8'))
+  module.exports = data
+} catch (err) {
+  const prefix = chalk.red('ERROR') + ` cannot read config file from ${chalk.blue(filename)}`
+  if (err.mark) {
+    // Markdown library error
```

Comment:
Yes, but they call it markdown in the YAML library (hence the err.mark thing). I'll make the comment clearer